app/_locales/en/messages.json

```
@@ -402,7 +402,7 @@
     "message": "You have 1 account connected to this site."
   },
   "connectedAccountsEmptyDescription": {
-    "message": "MetaMask is not connected this site. To connect to a web3 site, find the connect button on their site."
+    "message": "MetaMask is not connected to this page. If this is a web3 application, find and click the connect button."
```

[REQ] We appreciate your recommendations for copy changes! Definitely want to get that bug fixed, but we would like to keep the 'site' vernacular versus page, application, etc. 

```suggestion
    "message": "MetaMask is not connected to this site. To connect to a web3 site, find and click the connect button."
```